lib/ansible/modules/get_url.py

```diff
@@ -165,6 +165,13 @@
       - Header to identify as, generally appears in web server logs.
     type: str
     default: ansible-httpget
+  preserve_time:
+    description:
+      - Preserve "Last-Modified" datetime and set it as mtime of fetched file.
+      - If a "src" value is started "file:", it keeps mtime of the "src" file as mtime of the "dest" file.
```

Comment:
I question whether this should explicitly reference mtime?  `preserve_mtime`? `use_last_modified`?

Reply:
I thought this should be better the same behavior for both `http[s]:` and `file:`. In this case, `preserve_mtime` probably be suitable instead of `preserve_time` like you pointed out.
Or it would be better to modify it so that it only affects if users download a file via http[s]?
I would appreciate it if you give me an advice.

Reply:
Looking at this in an "old PRs" queue- it's really not the same as `preserve_mtime` options in other modules- `mtime_as_last_modified` maybe?